docs/SOPs/2-updating_lcia.md

```diff
@@ -0,0 +1,25 @@
+# Updating Life Cycle Impact Assessment Methods using LCIA Formatter
+
+## Purpose
+
+This SOP documents procedures for making updates or adding new LCIA methods to the Federal LCA Commons (FLCAC).
+
+## Procedure
+
+1. Any changes to an LCIA Method must align with a lciafmt package release and update to the package version.
```

Comment:
For someone not intimately familiar with the updating process, a little more context on this sentence would be helpful.